src/Reflow/Parser.hs

```diff
@@ -59,7 +59,7 @@ codeBlock = do
     s <- codeBlockChar
     c <- codeBlockContents
     e <- codeBlockChar
-    void eol
+    void (trailingWhitespace >> eol)
```

Comment:
I think `void $` would be more idiomatic. You could also push the `void` into `eol` and make it `Parser ()`. Do you ever care about the returned newline?